packages/near-fast-auth-signer-e2e-tests/utils/createAccount.ts

```diff
@@ -0,0 +1,116 @@
+import {
+  KeyPair,
+} from '@near-js/crypto';
+import { expect } from '@playwright/test';
+import admin from 'firebase-admin';
+import { sha256 } from 'js-sha256';
+import { CLAIM, getUserCredentialsFrpSignature } from 'near-fast-auth-signer/src/utils/mpc-service';
+
+import { serviceAccount } from './serviceAccount';
+
+const FIREBASE_API_KEY_TESTNET = 'AIzaSyDAh6lSSkEbpRekkGYdDM5jazV6IQnIZFU';
```

Comment:
Is this ok to be public?

Reply:
it has to be public.. 
First of all, if you look at `webpack.common.config.js` on packages/fast-auth-signer it's already declared there.
Secondly, it needs to be defined on it so that firebase API will point to execute. (If you inspect on network tab, you can easily notice this)

Reply:
Thanks for explaining, then no issues here :)